.env.appStore.example

```diff
@@ -20,6 +20,8 @@
 
 # - APP STORE **********************************************************************************************
 # ⚠️ ⚠️ ⚠️ THESE WILL BE MIGRATED TO THE DATABASE TO PREVENT AWS's 4KB ENV QUOTA ⚠️ ⚠️ ⚠️
+BASECAMP3_CLIENT_ID=
```

Comment:
Could you please add a title and description for these newly introduced environment variables? Also add the title to the index at the top. Furthermore, it will also need to point to a Cal.com README section on how to get the values. You can see the other environment values for other apps as examples.